content/docs/gke/deploy/deploy-cli.md

```diff
@@ -10,16 +10,25 @@ the deployment process and configuration than you get if you use the deployment
 UI. If you're looking for a simpler deployment procedure, see how to deploy 
 Kubeflow [using the deployment UI](/docs/gke/deploy/deploy-ui).
 
+## Before you start
+
 Before installing Kubeflow on the command line:
 
-  * Ensure you have installed the following tools:
-    
-     * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-     * [gcloud](https://cloud.google.com/sdk/)
+1. Ensure you have installed the following tools:
+  
+    * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+    * [gcloud](https://cloud.google.com/sdk/)
+
+1. If you're using
+  [Cloud Shell](https://cloud.google.com/shell/), enable 
+  [boost mode](https://cloud.google.com/shell/docs/features#boost_mode).
 
-  * If you're using
-    [Cloud Shell](https://cloud.google.com/shell/), enable 
-    [boost mode](https://cloud.google.com/shell/docs/features#boost_mode).
+1. If you want to use [Cloud Identity-Aware Proxy (Cloud 
+  IAP)](https://cloud.google.com/iap/docs/) for access control, follow the guide
+  to [setting up an OAuth credential](/docs/gke/deploy/oauth-setup). 
+  Cloud IAP is recommended for production deployments or deployments with 
```

Comment:
Avoid passive voice if possible.

Reply:
In this case, I think "is recommended" is OK. I can't think of a better alternative.